load/UpdateDBPFilesetTables.py

```diff
@@ -294,10 +294,10 @@ def getDuration(self, file):
 	from DBPLoadController import *
 
 	config = Config.shared()
-	lptsReader = LPTSExtractReader(config.filename_lpts_xml)
-	filesets = InputFileset.filesetCommandLineParser(config, lptsReader)
+	languageReader = LanguageReaderCreator().create(config)
```

Comment:
I think that we should import the LanguageReaderCreator in the main to execute the UpdateDBPFilesetTables.py file locally. e.g.

```python
from LanguageReaderCreator import LanguageReaderCreator
```